### ISSUE TITLE
Adjust scenarios to follow proper given-then-when patterns

### ISSUE THREAD

**Issue by robertoandrade** [testmunk/TMSample#2]
According to [Cumcumber docs](https://github.com/cucumber/cucumber/wiki/Given-When-Then), `Then` should only be used when expressing the desired outcome of the scenario instead of pre-conditions (which `Given`/`And` should be used instead), same applies for the actions being tested using `When`/`And`.

I see a lot of the scenarios described in this repo making the wrong use of `Then`, ie: in [my_first.feature](https://github.com/testmunk/TMSample/blob/master/testcases/features/my_first.feature):

```
Scenario: Login touch menu
  Given I log in
  Then I touch "Skip"
  Then I touch the "iconSlide" button
  Then I should see "Section One"
```

Would follow proper form if it was written as:

```
Scenario: Login touch menu
  Given I log in
  And I touch "Skip"
  When I touch the "iconSlide" button
  Then I should see "Section One"
```

Same applies to the [TMSampleAndroid](https://github.com/testmunk/TMSampleAndroid) scenarios.

**Comment by fryclukasz** [comment]
Thanks @robertoandrade, you are right here!

**Comment by lukasfr** [comment]
Fixed in a380b1562ab1e0cbec41f858dd9b409aba416214
